laws/src/main/scala/cats/laws/FunctorLaws.scala

```diff
@@ -33,6 +33,15 @@ trait FunctorLaws[F[_]] extends InvariantLaws[F] {
   def covariantIdentity[A](fa: F[A]): IsEq[F[A]] =
     fa.map(identity) <-> fa
 
+  def mapOrKeepEmpty[A](fa: F[A]): IsEq[F[A]] =
+    fa.mapOrKeep(PartialFunction.empty) <-> fa
+
+  def mapOrKeepIdentity[A](fa: F[A]): IsEq[F[A]] =
+    fa.mapOrKeep { case a => a } <-> fa
+
+  def mapOrKeepWiden[A, A1 >: A](fa: F[A]): IsEq[F[A1]] =
```

Comment:
I don't think we need this. This follows from the above and the law on widen.

Reply:
removed